src/services/highcharts-factory/highcharts-factory.svc.coffee

```diff
@@ -4,37 +4,22 @@ angular
 
   templates =
     line: Object.freeze
-      get: (series = [], options = {})->
-        zoomingOptions = _.get(options, 'withZooming')
-        xAxisOptions = if zoomingOptions?
-          {
-            events:
-              setExtremes: zoomingOptions.callback
-            max: _.get(zoomingOptions.defaults, 'max')
-            min: _.get(zoomingOptions.defaults, 'min')
-          }
-
         chart:
```

Comment:
@iseessel there is inconsistent indentation here. The `line` object properties are indented 4 spaces, and the rest of the file is 2 spaces.